### DISCUSSION TITLE
SBUS inav DYS F4 Pro V2

### DISCUSSION THREAD

**Discussion by rccrashcourse** [iNavFlight/inav#7548] [Q&A]
I use Frsky XM+, connected to DYS F4 Pro V2 with SBUS.
Works perfect with Betaflight, but it won't work with inav (same wiring, same settings).

SBUS connect on pad RX1
UART1 - Serial RX
Receiver tab: serial - SBUS
<img width="485" alt="Schermafbeelding 2021-05-24 om 14 54 14" src="https://user-images.githubusercontent.com/84719821/119350684-eb64b600-bc9f-11eb-91e1-1c9e997e4462.png">

Is this a bug or something I did't setup?

**Comment by tomekyo** [comment]
Hi, This is exactly my problem on a DYS F4 Pro. I had to change from HGLRC working fine and was fiddling around for ours troubleshooting it. Using oscilloscope I was able to verify that the SBUS signal definitely goes into the FC and though the inverter it is being fed into the processor. One way or another the INAV doesn't seem to detect it. I flashed back to BF and the receiver is working. Why would you bother to publish the code for the board which does not support it...?
